.github/workflows/tests.yaml

```diff
@@ -22,7 +22,6 @@ jobs:
 
       - name: Run simulation notebooks
         run: |
-          pytest -v --nbval-lax --current-env tests/train_vae_test.ipynb --cov=ponyo
           pytest -v --nbval-lax --current-env tests/simulation_tests.ipynb --cov=ponyo --cov-append
```

Comment:
just curious -- what does `--cov-append` change?

Reply:
So this is saying to accumulate the statistics from each pytest run so that coveralls can calculate the overall coverage of the test scripts